sample_configuration.properties

```
@@ -2,17 +2,17 @@ api.url={API_URL}
 database.url={DATABASE_URL}
 database.username={DATABASE_USERNAME}
 database.password={DATABASE_PASSWORD}
-afet_yonetim_sistemi_admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
-afet_yonetim_sistemi_admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
-volunteer_foundation_admin.email_address={VOLUNTEER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
-volunteer_foundation_admin.password={VOLUNTEER_FOUNDATION_ADMIN_PASSWORD}
-disaster_foundation_admin.email_address={DISASTER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
-disaster_foundation_admin.password={DISASTER_FOUNDATION_ADMIN_PASSWORD}
-test_foundation_admin.email_address={TEST_FOUNDATION_ADMIN_EMAIL_ADDRESS}
-test_foundation_admin.password={TEST_FOUNDATION_ADMIN_PASSWORD}
-landing_page_user_martha.email_address={LANDING_PAGE_USER_MARTHA_EMAIL_ADDRESS}
-landing_page_user_martha.password={LANDING_PAGE_USER_MARTHA_PASSWORD}
-volunteer_foundation_id={VOLUNTEER_FOUNDATION_ID}
-afet_yonetim_sistemi_id={AFET_YONETIM_SISTEMI_ID}
-disaster_foundation_id={DISASTER_FOUNDATION_ID}
-test_foundation_id={TEST_FOUNDATION_ID}
\ No newline at end of file
+afet_yonetim_sistemi.admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
+afet_yonetim_sistemi.admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
+volunteer_foundation.admin.email_address={VOLUNTEER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+volunteer_foundation.admin.password={VOLUNTEER_FOUNDATION_ADMIN_PASSWORD}
+disaster_foundation.admin.email_address={DISASTER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+disaster_foundation.admin.password={DISASTER_FOUNDATION_ADMIN_PASSWORD}
+test_foundation.admin.email_address={TEST_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+test_foundation.admin.password={TEST_FOUNDATION_ADMIN_PASSWORD}
+disaster_foundation.user.email_address={DISASTER_FOUNDATION_USER_EMAIL_ADDRESS}
+disaster_foundation.user.password={DISASTER_FOUNDATION_USER_PASSWORD}
+volunteer_foundation.id={VOLUNTEER_FOUNDATION_ID}
+afet_yonetim_sistemi.id={AFET_YONETIM_SISTEMI_ID}
+disaster_foundation.id={DISASTER_FOUNDATION_ID}
+test_foundation.id={TEST_FOUNDATION_ID}
```

Düzen açısından aşağıdaki gibi olması daha iyi olabilir.
```suggestion
afet_yonetim_sistemi.id={AFET_YONETIM_SISTEMI_ID}
afet_yonetim_sistemi.admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
afet_yonetim_sistemi.admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
volunteer_foundation.id={VOLUNTEER_FOUNDATION_ID}
volunteer_foundation.admin.email_address={VOLUNTEER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
volunteer_foundation.admin.password={VOLUNTEER_FOUNDATION_ADMIN_PASSWORD}
disaster_foundation.id={DISASTER_FOUNDATION_ID}
disaster_foundation.admin.email_address={DISASTER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
disaster_foundation.admin.password={DISASTER_FOUNDATION_ADMIN_PASSWORD}
disaster_foundation.user.email_address={DISASTER_FOUNDATION_USER_EMAIL_ADDRESS}
disaster_foundation.user.password={DISASTER_FOUNDATION_USER_PASSWORD}
test_foundation.id={TEST_FOUNDATION_ID}
test_foundation.admin.email_address={TEST_FOUNDATION_ADMIN_EMAIL_ADDRESS}
test_foundation.admin.password={TEST_FOUNDATION_ADMIN_PASSWORD}

```
Ek olarak mümkünse sadece 1 ya da 2 kurum kullanalım. Bütün kurumları kullanmak yerine daha sağlıklı olabilir.